rust/dev-pl011/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+# Rust support for LK, main package
+
+[package]
+name = "dev-pl011"
+version = "0.1.0"
+edition = "2024"
+description = """
+Rust PL011 uart driver for LK.
+"""
+
+[dependencies.lk]
+version = "0.1.0"
+path = "../lk"
+
+[dependencies.log]
```

Comment:
Question: what are the licenses of these two libs. Are they compatible with LKs?

Reply:
Look more deeply, the LICENSE file at the top _is_ actually the MIT text. So, I've put MIT headers on each of the files I've created (the ones from trusty seem to already have that), and added `license = "MIT"` lines to the Cargo.toml files.